docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -48,9 +48,22 @@ const theme = createTheme({
 
 {{"demo": "GlobalThemeOverride.js"}}
 
-The list of each component's classes is documented under the **CSS** section of its API page.
+Note that each component is composed of several different parts. If you want to know how to target a specific part of a given component, check the **CSS** section of its API page to see every class it has available. You should then use it inside the `stylesOverrides` key.
 
-To override a lab component's styles with TypeScript, check [this section of the documentation](/components/about-the-lab/#typescript).
+```js
+const theme = createTheme({
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        containedPrimary: {
```

Comment:
@siriwatknp as we are going in the direction of removing these, should we maybe change this demo a bit?

Reply:
Good point, will do it.